app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -426,4 +429,13 @@ class BrowserNavigationOverlayScrollView(
         val deltaScrollForOnScreen = super.computeScrollDeltaToGetChildRectOnScreen(rect)
         return deltaScrollForOnScreen + deltaScrollPadding * Integer.signum(deltaScrollForOnScreen)
     }
+
+    fun updateOverlayForHomescreen(isHomescreen: Boolean) {
+        val windowSpacerHeight = if (isHomescreen) OVERLAY_SPACER_HOMESCREEN_HEIGHT else OVERLAY_SPACER_WEBRENDER_HEIGHT
+        overlayWindowSpacer.apply {
+            layoutParams.height = windowSpacerHeight
+            requestLayout()
+        }
```

Comment:
I feel like this is something that should go through our standard architecture rather than being called directly by the `ScreenController`.  We can add a very small `OverlayViewModel` with one field, `viewIsSplit`.  It can observe the `SessionRepo` to get the current URL, and `BrowserNavigationOverlayScrollView` can observe it to set the spacer height.

It sounds like a pain, but the actual amount of code should be pretty small.